src/index.html

```diff
@@ -2,12 +2,51 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="product-card__image"
+        width="160"
+        height="134"
+      >
+      <h2 class="product-card__title">
+        APPLE A1419 iMac 27" Retina
+        5K Monoblock (MNED2UA/A)
+      </h2>
+      <div class="product-card__product-code">Product code: 195434</div>
+      <div class="product-card__block product-card__block--upper">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="product-card__reviews">Reviews: 5</div>
+      </div>
+      <div class="product-card__block product-card__block--lower">
+        <div class="product-card__price-title">Price:</div>
+        <div class="product-card__price-amount">$2,199</div>
+      </div>
+      <a
```

Comment:
```suggestion
      </div>
      
      <div class="product-card__block product-card__block--lower">
        <div class="product-card__price-title">Price:</div>
        <div class="product-card__price-amount">$2,199</div>
      </div>
      
      <a
```
You can use semantic tags here too